internal/gh/project_next_fields_test.go

```diff
@@ -0,0 +1,134 @@
+package gh
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetIterationTitleByID(t *testing.T) {
+	activeMany := `{"configuration":{"duration":14,"start_day":1, "iterations":
+[{"id":"5f065395","title":"Sprint 1","duration":14,"start_date":"2022-06-20","title_html":"Sprint 1"},
+{"id":"a0f5e3ae","title":"Sprint 2","duration":14,"start_date":"2022-07-04","title_html":"Sprint 2"},
+{"id":"fd9fd832","title":"Sprint 3","duration":14,"start_date":"2022-07-18","title_html":"Sprint 3"}],
+"completed_iterations":[]}}`
+	activeOne := `{"configuration":{"duration":14,"start_day":1,"iterations":
+[{"id":"5f065395","title":"Sprint 1","duration":14,"start_date":"2022-06-20","title_html":"Sprint 1"}],
+"completed_iterations":
+[{"id":"a0f5e3ae","title":"Sprint 2","duration":14,"start_date":"2022-07-04","title_html":"Sprint 2"},
+{"id":"fd9fd832","title":"Sprint 3","duration":14,"start_date":"2022-07-18","title_html":"Sprint 3"}]}}`
+
+	tc := []struct {
+		name          string
+		id            string
+		settings      string
+		expectedTitle string
+		expectErr     bool
```

Comment:
also would you please rename `expectErr` to `expectedErr`?

Reply:
fixed